sample_worksheet.py

```diff
@@ -0,0 +1,338 @@
+        self._add_bundles(self._valid_bundles)
+        self._add_subheader('Private Bundle References')
+        self._add_bundles(self._private_bundles)
+
+    def _add_schemas(self):
+        self._add_header('Schemas')
+        self._add_subheader('Valid Schema')
+        self._add_text('% schema valid_schema')
+        self._add_text('% add uuid uuid "[0:8]"')
+        self._add_text('% add name')
+        self._add_text('% add summary')
+        self._add_text('% add metadata')
+        self._add_text('% add permission')
+        self._add_text('% add group_permissions')
+        self._add_text('% display table valid_schema')
+        self._add_bundles(self._valid_bundles)
+
+        self._add_description('Attempting to reference private bundles with a valid schema')
+        self._add_text('% display table valid_schema')
+        self._add_bundles(self._private_bundles)
+
+        self._add_subheader('Post-Processor Schema')
+        self._add_text('% schema post_processor_schema')
+        self._add_text('% add "duration time" time duration')
+        self._add_text('% add updated last_updated date')
+        self._add_text('% add size data_size size')
+        self._add_text('% add uuid uuid "[0:8]"')
+        self._add_text('% display table post_processor_schema')
+        self._add_bundles(self._valid_bundles)
+
+        self._add_subheader('Combine Schemas')
+        self._add_text('% schema combined_schema')
+        self._add_text('% addschema valid_schema')
+        self._add_text('% addschema post_processor_schema')
+        self._add_text('% display table combined_schema')
+        self._add_bundles(self._valid_bundles)
+
+        self._add_subheader('Invalid Schemas')
+        self._add_description('Attempting to add a field before referencing a schema')
+        self._add_text('% add name')
+        self._add_description('Attempting to add a non-existing schema')
+        self._add_text('% schema invalid_schema')
+        self._add_text('% addschema nonexistent_schema')
+        self._add_description('Attempting to create a schema with invalid functions')
+        self._add_text('% schema invalid_functions_schema')
+        self._add_text('% add time time duration2')
+        self._add_text('% add updated last_updated date2')
+        self._add_text('% add size data_size size2')
+        self._add_text('% display table invalid_functions_schema')
+        self._add_bundles(self._valid_bundles)
+
+    def _add_display_modes(self):
+        self._add_header('Display Modes')
+        self._add_subheader('Table')
+        self._add_text('% display table valid_schema')
+        self._add_bundles(self._valid_bundles)
+
+        self._add_subheader('Image')
+        for uuid in self._search_bundles('.png'):
+            self._add_text('% display image / width=500')
+            self._add_bundle(uuid)
+
+        self._add_subheader('Record')
+        self._add_text('% display record valid_schema')
```

Comment:
Line 175 and 176 causes the page to crash with React errors. I have to comment these two lines out for now when creating the worksheet.

I filed a separate issue to investigate: https://github.com/codalab/codalab-worksheets/issues/1885